### YamlMime:PythonClass
uid: msrestazure.azure_operation.BadResponse
name: BadResponse
fullName: msrestazure.azure_operation.BadResponse
module: msrestazure.azure_operation
inheritances:
- builtins.Exception
constructor:
  syntax: BadResponse()
methods:
- uid: msrestazure.azure_operation.BadResponse.add_note
  name: add_note
  summary: "Exception.add_note(note) \u2013\nadd a note to the exception"
  signature: add_note()
- uid: msrestazure.azure_operation.BadResponse.with_traceback
  name: with_traceback
  summary: "Exception.with_traceback(tb) \u2013\nset self.__traceback__ to tb and\
    \ return self."
  signature: with_traceback()
attributes:
- uid: msrestazure.azure_operation.BadResponse.args
  name: args
